canonical-websites/www.ubuntu.com
# Redirect the most common 404 pages - part 1

## Done

* Redirect the 5 most common `404` pages on ubuntu.com
* This addresses 20% of `404` errors throughout the site

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8001/](http://0.0.0.0:8001/)
- Run through the following [QA steps](https://github.com/canonical-webteam/practices/blob/master/workflow/qa-steps.md)
- Ensure the following URLs are redirected to a working page:
  * [/18.04](http://www.ubuntu.com-pr-3916.run.demo.haus/18.04)
  * [/community/participate](http://www.ubuntu.com-pr-3916.run.demo.haus/community/participate)
  * [/GetUbuntu/download](http://www.ubuntu.com-pr-3916.run.demo.haus/GetUbuntu/download)
  * [/privacy-policy/third-parties](http://www.ubuntu.com-pr-3916.run.demo.haus/privacy-policy/third-parties)
  * [/start-download?distro=desktop&bits=64&release=lts](http://www.ubuntu.com-pr-3916.run.demo.haus/start-download?distro=desktop&bits=64&release=lts)
